testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/webauthn/pages/fragments/LoggedInPageHeader.java

```diff
@@ -27,40 +27,24 @@
  * @author Vaclav Muzikar <vmuzikar@redhat.com>
  */
 public class LoggedInPageHeader extends AbstractHeader {
-    @FindBy(xpath = "//*[@data-testid='options']//a[text() = 'Sign out']")
+    @FindBy(xpath = "//div[@id='app']//header[@class='pf-v5-c-page__header']//span[text() = 'Sign out']")
```

Comment:
This selector seems very specific, since there is only ever a single page header we can probably get away with:
```suggestion
    @FindBy(xpath = "//[@class='pf-v5-c-page__header']//[text() = 'Sign out']")
```

Not sure if this is a valid XPath selector, but it's generally good practice to make selectors less specific, especially when matching tag names (e.g. `div`, `span`) because these can change easily.